react-native/react/tracker/proofs.render.desktop.js

```diff
@@ -48,6 +48,7 @@ export default class ProofsRender extends Component {
 
     const statusColor = {
       normal: colors.lightBlue,
+      loggedOut: colors.lightBlue,
```

Comment:
sounds good! This missing property was detected by flow! so that's pretty cool :)